### ISSUE TITLE
Presentation

### ISSUE THREAD

**Issue by Pranjal-Srivastava-2023** [ubsuny/23-Homework6G3#65]
Do we have presentation for this HW? Who will be presenting?

**Comment by reshnashrestha** [comment]
We don't have any classes. So let's wait for the response from prof.

**Comment by AhmedCode99** [comment]
I think we may be presenting on monday

**Comment by AhmedCode99** [comment]
You should ask Dr. Tim on discord

**Comment by Pranjal-Srivastava-2023** [comment]
Yes, we might be presenting on Monday.

**Comment by reshnashrestha** [comment]
Oh then it might be like those who haven’t got chance to present will have chances, and it could be task wise. 
@laserlab will there be any presentation on this hw?

**Comment by Pranjal-Srivastava-2023** [comment]
I guess we will each present our tasks.

**Comment by laserlab** [comment]
Yes those who have less than 2 presentations can present on Monday

**Comment by reshnashrestha** [comment]
Thank you for informing

**Comment by LinxuanHu** [comment]
i think it will be on monday

**Comment by s4il3sh** [comment]
so who will be presenting?
